spec/models/miq_report_spec.rb

```diff
@@ -552,6 +566,9 @@ def generate_html_row(is_even, tenant_name, formatted_values)
         formatted_values = {:name => "Allocated Number of Templates", :total => "4 Count", :used => "1 Count",
                             :allocated => "0 Count", :available => "3 Count"}
         @expected_html_rows.push(generate_html_row(true, @tenant.name, formatted_values))
+
+        EvmSpecHelper.local_miq_server
```

Comment:
@lpichler Why was this line needed?  Does it generate_table need it because it wants the server timezone?

If so, can we move this to the test that needs it?